systers_portal/templates/account/signup.html

```diff
@@ -0,0 +1,63 @@
+{% extends "account/base.html" %}
+{% load url from future %}
```

Comment:
The syntax `{% load _ from future %}` is deprecated and will be removed in Django 1.9.

Proof link - [Django 1.7 release notes](https://docs.djangoproject.com/en/1.7/releases/1.7/#loading-ssi-and-url-template-tags-from-future-library).